addons/sale/migrations/10.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -14,7 +23,6 @@ sale         / sale.layout_category     / subtotal (boolean)            : NEW
 # NOTHING TO DO
```

Comment:
This is not correct, as sale now integrates sale_layout module. You need to run update_module_names with the merge option activated and check if there's any change on data model